<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form{
    background-color: rgb(255, 255, 255);
}

#pushButton {
	background-color: rgb(85, 170, 255);
	color: rgb(255, 255, 255);
    border-radius: 2px;
}

#pushButton:pressed {
    background-color:rgb(125, 190, 255); 
    border-style: inset;
}

QListWidget,QTextEdit,QPushButton {
    border: 0px solid;
}

#listWidget {
	background-color: rgb(245, 245, 245);
}

#listWidget::Item:hover{
	background-color: rgb(208, 208, 208);
}

#listWidget_2 {
   border-bottom : 1px solid #f1f1f1;
}

#listWidget,#listWidget_2 {
    outline:0px;
}

#listWidget::item:selected{
    background-color:rgb(226, 226, 226);
}

#listWidget_2::Item:hover, #listWidget_2::item:selected{
	border: 0px solid;
}

QScrollBar:vertical {              
    width:10px;
    margin: 0px;
    background:transparent;
    padding:0px;
}

QScrollBar::handle:vertical {
	background:rgba(0,0,0,15%);
	border-radius:5px;
}

QScrollBar::add-line:vertical {
    background:none;
}

QScrollBar::sub-line:vertical {
    background:none;
}

QScrollBar::handle:vertical:hover, QScrollBar::handle:vertical:pressed {
    background:rgba(0,0,0,30%);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QQTitlePanel" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,6,0,2,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QQAvatarPanel" name="widget_3" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_2">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QQChatPanel" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="rightMargin">
          <number>5</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>发送(&amp;S)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQChatPanel</class>
   <extends>QWidget</extends>
   <header>qqchatpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QQTitlePanel</class>
   <extends>QWidget</extends>
   <header>qqtitlepanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QQAvatarPanel</class>
   <extends>QWidget</extends>
   <header>qqavatarpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
